### PR TITLE
ps3netsrv: location with whitespaces fix #3399

### LINKED ISSUE #3399: ps3netserv not starting on DS916 DSM6.1
_For new Package Requests, see the guidelines_  

### Setup
_Package Name:_ PS3netsrv
_Package Version:_ 1.47.08-2

_NAS Model:_ DS916
_NAS Architecture:_ Braswell
_DSM version:_ DSM 6.1.7-15284 Update 2

### Expected behavior
Package should start

### Actual behavior
Package did not start

### Steps to reproduce
_1._ Install package
_2._ Run package
_3._ Package does not run

#### Package log
_Check Package Center or `/usr/local/{package}/var/`_
```
Thu Jul 26 00:45:12 CEST 2018
Starting ps3netsrv command /bin/stdbuf -o L -e L /volume1/@appstore/ps3netsrv/bin/ps3netsrv
/volume1/@appstore/ps3netsrv/bin/ps3netsrv: /lib/libstdc++.so.6: no version information available (required by /volume1/@appstore/ps3netsrv/bin/ps3netsrv)
/volume1/@appstore/ps3netsrv/bin/ps3netsrv: /lib/libstdc++.so.6: no version information available (required by /volume1/@appstore/ps3netsrv/bin/ps3netsrv)
ps3netsrv build 20170310 (mod by aldostools)

Usage: ps3netsrv [rootdirectory] [port] [whitelist]

 Default port: 38008

 Whitelist: x.x.x.x, where x is 0-255 or *
 (e.g 192.168.1.* to allow only connections from 192.168.1.0-192.168.1.255)

```
#### Other logs
_E.g. `/var/log/messages` or `/var/log/synopkg.log`_
```
/var/log/messages/:

2018-07-25T01:16:08+02:00 Diskstation916 kernel: [709303.748637] net_ratelimit: 3 callbacks suppressed
2018-07-25T16:58:45+02:00 Diskstation916 builtin-dyn-myds-job: myds_report_info.cpp:118 Failed to query service check, resp={"code":"badparam"}
2018-07-25T16:58:45+02:00 Diskstation916 builtin-dyn-myds-job: synomyds.cpp:137 Failed to report MyDS info, code=1
2018-07-25T22:25:28+02:00 Diskstation916 synoscgi_SYNO.Core.Package.Installation_1_install[8906]: resource_api.cpp:163 Acquire service-cfg for ps3netsrv when 0x0001 (done)
2018-07-25T22:33:18+02:00 Diskstation916 synoscgi_SYNO.Core.Package.Uninstallation_1_uninstall[12434]: resource_api.cpp:231 Release service-cfg for ps3netsrv when 0x0004 (done)
2018-07-25T22:33:21+02:00 Diskstation916 synodrnode: Delete session of user[sc-ps3netsrv] by user-delete hook
2018-07-25T22:33:49+02:00 Diskstation916 synoscgi_SYNO.Core.Package.Installation_1_install[13247]: resource_api.cpp:163 Acquire service-cfg for ps3netsrv when 0x0001 (done)
2018-07-25T22:34:15+02:00 Diskstation916 synoscgi_SYNO.Docker.Registry_1_tags[13946]: util.cpp:296 Fail to query url=https://hub.docker.com/v2/repositories/library/mccloud/ps3netsrv/tags/?page=1&page_size=100, CURLcode=22, err=Error
2018-07-25T22:34:15+02:00 Diskstation916 synoscgi_SYNO.Docker.Registry_1_tags[13946]: util.cpp:298 httpCode=404
2018-07-25T22:34:25+02:00 Diskstation916 synoscgi_SYNO.Docker.Registry_1_tags[14023]: util.cpp:296 Fail to query url=https://hub.docker.com/v2/repositories/library/dreamcat4/ps3netsrv/tags/?page=1&page_size=100, CURLcode=22, err=Error
2018-07-25T22:34:25+02:00 Diskstation916 synoscgi_SYNO.Docker.Registry_1_tags[14023]: util.cpp:298 httpCode=404
2018-07-25T22:34:39+02:00 Diskstation916 synoscgi_SYNO.Core.Package.Uninstallation_1_uninstall[14121]: resource_api.cpp:231 Release service-cfg for ps3netsrv when 0x0004 (done)
2018-07-25T22:34:43+02:00 Diskstation916 synodrnode: Delete session of user[sc-ps3netsrv] by user-delete hook
2018-07-25T22:34:59+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[14029]: mail_read_by_tag_and_convert.c (656) Bad email format.
Onderwerp: Docker-image: downloaden van docker.io/dreamcat4/ps3netsrv is voltooid.
2018-07-25T22:34:59+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[14029]: mail_read_by_tag_and_convert.c (766) Failed to get subject and content from temp file.
2018-07-25T22:34:59+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[14029]: sysnotify_send_notification.c:222 Failed to get mail content.
2018-07-25T22:34:59+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[14029]: image.cpp:982 Fail to send notification.
2018-07-25T22:37:12+02:00 Diskstation916 kernel: [786167.033335] aufs au_opts_verify:1571:dockerd[18734]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-25T22:37:13+02:00 Diskstation916 kernel: [786167.520081] aufs au_opts_verify:1571:dockerd[18734]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-25T22:37:16+02:00 Diskstation916 kernel: [786170.673474] aufs au_opts_verify:1571:dockerd[19660]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-25T22:37:20+02:00 Diskstation916 ntpd[8140]: bind(34) AF_INET6 fe80::d80c:5dff:fec8:cbf8%56#123 flags 0x11 failed: Cannot assign requested address
2018-07-25T22:37:20+02:00 Diskstation916 ntpd[8140]: unable to create socket on docker4478e79 (35) for fe80::d80c:5dff:fec8:cbf8%56#123
2018-07-25T22:58:26+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-25T22:58:26+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-25T22:58:45+02:00 Diskstation916 synonetd: SYSTEM:  Last message 'net_if_is_bonding_sl' repeated 1 times, suppressed by syslog-ng on Diskstation916
2018-07-25T22:58:45+02:00 Diskstation916 kernel: [787459.829672] aufs au_opts_verify:1571:dockerd[18768]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-25T22:58:50+02:00 Diskstation916 kernel: [787464.422800] aufs au_opts_verify:1571:dockerd[18888]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-25T23:25:25+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-25T23:25:25+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-25T23:25:25+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-25T23:42:07+02:00 Diskstation916 kernel: [790061.943220] aufs au_opts_verify:1571:dockerd[20616]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-25T23:43:34+02:00 Diskstation916 kernel: [790148.265754] aufs au_opts_verify:1571:dockerd[20314]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-25T23:43:38+02:00 Diskstation916 ntpd[8140]: bind(34) AF_INET6 fe80::202e:17ff:fe07:c52f%60#123 flags 0x11 failed: Cannot assign requested address
2018-07-25T23:43:38+02:00 Diskstation916 ntpd[8140]: unable to create socket on docker5b1adf2 (38) for fe80::202e:17ff:fe07:c52f%60#123
2018-07-25T23:50:10+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-25T23:50:10+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-25T23:50:22+02:00 Diskstation916 synonetd: SYSTEM:  Last message 'net_if_is_bonding_sl' repeated 2 times, suppressed by syslog-ng on Diskstation916
2018-07-25T23:50:22+02:00 Diskstation916 synoscgi_SYNO.Docker.Registry_1_tags[8658]: util.cpp:296 Fail to query url=https://hub.docker.com/v2/repositories/library/mccloud/ps3netsrv/tags/?page=1&page_size=100, CURLcode=22, err=Error
2018-07-25T23:50:22+02:00 Diskstation916 synoscgi_SYNO.Docker.Registry_1_tags[8658]: util.cpp:298 httpCode=404
2018-07-25T23:51:00+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[8907]: mail_read_by_tag_and_convert.c (656) Bad email format.
Onderwerp: Docker-image: downloaden van docker.io/mccloud/ps3netsrv is voltooid.
2018-07-25T23:51:00+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[8907]: mail_read_by_tag_and_convert.c (766) Failed to get subject and content from temp file.
2018-07-25T23:51:00+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[8907]: sysnotify_send_notification.c:222 Failed to get mail content.
2018-07-25T23:51:00+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[8907]: image.cpp:982 Fail to send notification.
2018-07-25T23:53:11+02:00 Diskstation916 kernel: [790725.580770] aufs au_opts_verify:1571:dockerd[20616]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-25T23:53:11+02:00 Diskstation916 kernel: [790725.922301] aufs au_opts_verify:1571:dockerd[20616]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-25T23:53:12+02:00 Diskstation916 kernel: [790727.029547] aufs au_opts_verify:1571:dockerd[20616]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-25T23:59:19+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-25T23:59:19+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-25T23:59:19+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-26T00:00:20+02:00 Diskstation916 kernel: [791155.157014] aufs au_opts_verify:1571:dockerd[19660]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-26T00:00:23+02:00 Diskstation916 kernel: [791157.598275] aufs au_opts_verify:1571:dockerd[20299]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-26T00:02:59+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-26T00:02:59+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-26T00:03:40+02:00 Diskstation916 synonetd: SYSTEM:  Last message 'net_if_is_bonding_sl' repeated 1 times, suppressed by syslog-ng on Diskstation916
2018-07-26T00:03:40+02:00 Diskstation916 kernel: [791355.131677] aufs au_opts_verify:1571:dockerd[21883]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-26T00:03:42+02:00 Diskstation916 kernel: [791357.005693] aufs au_opts_verify:1571:dockerd[21883]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-26T00:06:02+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-26T00:06:02+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-26T00:06:42+02:00 Diskstation916 synonetd: SYSTEM:  Last message 'net_if_is_bonding_sl' repeated 1 times, suppressed by syslog-ng on Diskstation916
2018-07-26T00:06:42+02:00 Diskstation916 synoscgi_SYNO.Docker.Registry_1_tags[16925]: util.cpp:296 Fail to query url=https://hub.docker.com/v2/repositories/library/shawly/ps3netsrv/tags/?page=1&page_size=100, CURLcode=22, err=Error
2018-07-26T00:06:42+02:00 Diskstation916 synoscgi_SYNO.Docker.Registry_1_tags[16925]: util.cpp:298 httpCode=404
2018-07-26T00:07:03+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[17007]: mail_read_by_tag_and_convert.c (656) Bad email format.
Onderwerp: Docker-image: downloaden van docker.io/shawly/ps3netsrv is voltooid.
2018-07-26T00:07:03+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[17007]: mail_read_by_tag_and_convert.c (766) Failed to get subject and content from temp file.
2018-07-26T00:07:03+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[17007]: sysnotify_send_notification.c:222 Failed to get mail content.
2018-07-26T00:07:03+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[17007]: image.cpp:982 Fail to send notification.
2018-07-26T00:08:30+02:00 Diskstation916 kernel: [791644.732892] aufs au_opts_verify:1571:dockerd[20299]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-26T00:08:30+02:00 Diskstation916 kernel: [791645.016823] aufs au_opts_verify:1571:dockerd[20299]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-26T00:08:58+02:00 Diskstation916 kernel: [791672.555088] aufs au_opts_verify:1571:dockerd[19663]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-26T00:09:01+02:00 Diskstation916 kernel: [791676.184231] aufs au_opts_verify:1571:dockerd[21883]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-26T00:10:50+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-26T00:10:50+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-26T00:11:03+02:00 Diskstation916 synonetd: SYSTEM:  Last message 'net_if_is_bonding_sl' repeated 1 times, suppressed by syslog-ng on Diskstation916
2018-07-26T00:11:03+02:00 Diskstation916 kernel: [791797.371049] aufs au_opts_verify:1571:dockerd[18888]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-26T00:11:25+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-26T00:11:25+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-26T00:11:51+02:00 Diskstation916 synonetd: SYSTEM:  Last message 'net_if_is_bonding_sl' repeated 1 times, suppressed by syslog-ng on Diskstation916
2018-07-26T00:11:51+02:00 Diskstation916 synoscgi_SYNO.Docker.Container_1_get_process[21479]: profile.cpp:352 [/var/packages/Docker/etc/shawly-ps3netsrv1.config] does not exist.
2018-07-26T00:11:51+02:00 Diskstation916 synoscgi_SYNO.Docker.Container_1_get[21480]: profile.cpp:352 [/var/packages/Docker/etc/shawly-ps3netsrv1.config] does not exist.
2018-07-26T00:11:51+02:00 Diskstation916 synoscgi_SYNO.Docker.Container_1_get[21480]: container.cpp:733 Error: The container name is empty
2018-07-26T00:12:21+02:00 Diskstation916 synoscgi_SYNO.Docker.Registry_1_tags[21771]: util.cpp:296 Fail to query url=https://hub.docker.com/v2/repositories/library/xioustic/ps3netsrv/tags/?page=1&page_size=100, CURLcode=22, err=Error
2018-07-26T00:12:21+02:00 Diskstation916 synoscgi_SYNO.Docker.Registry_1_tags[21771]: util.cpp:298 httpCode=404
2018-07-26T00:12:36+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[21776]: mail_read_by_tag_and_convert.c (656) Bad email format.
Onderwerp: Docker-image: downloaden van docker.io/xioustic/ps3netsrv is voltooid.
2018-07-26T00:12:36+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[21776]: mail_read_by_tag_and_convert.c (766) Failed to get subject and content from temp file.
2018-07-26T00:12:36+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[21776]: sysnotify_send_notification.c:222 Failed to get mail content.
2018-07-26T00:12:36+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[21776]: image.cpp:982 Fail to send notification.
2018-07-26T00:13:26+02:00 Diskstation916 kernel: [791941.032597] aufs au_opts_verify:1571:dockerd[12635]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-26T00:13:27+02:00 Diskstation916 kernel: [791941.370502] aufs au_opts_verify:1571:dockerd[12635]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-26T00:13:28+02:00 Diskstation916 kernel: [791942.481838] aufs au_opts_verify:1571:dockerd[19660]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-26T00:16:26+02:00 Diskstation916 synoscgi_SYNO.Docker.Registry_1_tags[23262]: util.cpp:296 Fail to query url=https://hub.docker.com/v2/repositories/library/mccloud/ps3netsrv/tags/?page=1&page_size=100, CURLcode=22, err=Error
2018-07-26T00:16:26+02:00 Diskstation916 synoscgi_SYNO.Docker.Registry_1_tags[23262]: util.cpp:298 httpCode=404
2018-07-26T00:16:30+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-26T00:16:30+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-26T00:16:41+02:00 Diskstation916 synonetd: SYSTEM:  Last message 'net_if_is_bonding_sl' repeated 1 times, suppressed by syslog-ng on Diskstation916
2018-07-26T00:16:41+02:00 Diskstation916 synoscgi_SYNO.Docker.Container_1_get[23546]: profile.cpp:352 [/var/packages/Docker/etc/xioustic-ps3netsrv1.config] does not exist.
2018-07-26T00:16:41+02:00 Diskstation916 synoscgi_SYNO.Docker.Container_1_get[23546]: container.cpp:733 Error: The container name is empty
2018-07-26T00:16:44+02:00 Diskstation916 synoscgi_SYNO.Docker.Container_1_get_process[23553]: profile.cpp:352 [/var/packages/Docker/etc/xioustic-ps3netsrv1.config] does not exist.
2018-07-26T00:17:06+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[23312]: mail_read_by_tag_and_convert.c (656) Bad email format.
Onderwerp: Docker-image: downloaden van docker.io/mccloud/ps3netsrv is voltooid.
2018-07-26T00:17:06+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[23312]: mail_read_by_tag_and_convert.c (766) Failed to get subject and content from temp file.
2018-07-26T00:17:06+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[23312]: sysnotify_send_notification.c:222 Failed to get mail content.
2018-07-26T00:17:06+02:00 Diskstation916 synoscgi_SYNO.Docker.Image_1_pull_start[23312]: image.cpp:982 Fail to send notification.
2018-07-26T00:18:23+02:00 Diskstation916 kernel: [792237.970740] aufs au_opts_verify:1571:dockerd[12634]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-26T00:18:24+02:00 Diskstation916 kernel: [792238.278272] aufs au_opts_verify:1571:dockerd[12634]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-26T00:18:24+02:00 Diskstation916 kernel: [792239.204789] aufs au_opts_verify:1571:dockerd[12635]: dirperm1 breaks the protection by the permission bits on the lower branch
2018-07-26T00:26:21+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-26T00:26:21+02:00 Diskstation916 synonetd: net_if_is_bonding_slave.c:46 Cannot get flags of network interface: No such device
2018-07-26T00:26:35+02:00 Diskstation916 synonetd: SYSTEM:  Last message 'net_if_is_bonding_sl' repeated 1 times, suppressed by syslog-ng on Diskstation916
2018-07-26T00:26:35+02:00 Diskstation916 synoscgi_SYNO.Core.Package.Installation_1_install[27281]: resource_api.cpp:163 Acquire service-cfg for ps3netsrv when 0x0001 (done)
2018-07-26T00:36:10+02:00 Diskstation916 synoscgi_SYNO.Core.Package.Uninstallation_1_uninstall[30223]: resource_api.cpp:231 Release service-cfg for ps3netsrv when 0x0004 (done)
2018-07-26T00:36:14+02:00 Diskstation916 synodrnode: Delete session of user[sc-ps3netsrv] by user-delete hook
2018-07-26T00:37:06+02:00 Diskstation916 synoscgi_SYNO.Core.Package.Installation_1_install[31231]: resource_api.cpp:163 Acquire service-cfg for ps3netsrv when 0x0001 (done)
2018-07-26T00:42:43+02:00 Diskstation916 synoscgi_SYNO.Core.Package.Uninstallation_1_uninstall[1063]: resource_api.cpp:231 Release service-cfg for ps3netsrv when 0x0004 (done)
2018-07-26T00:42:46+02:00 Diskstation916 synodrnode: Delete session of user[sc-ps3netsrv] by user-delete hook
2018-07-26T00:44:51+02:00 Diskstation916 synoscgi_SYNO.Core.Package.Installation_1_install[2145]: resource_api.cpp:163 Acquire service-cfg for ps3netsrv when 0x0001 (done)
2018-07-26T01:03:38+02:00 Diskstation916 synoscgi_SYNO.Core.Package.Uninstallation_1_uninstall[7349]: resource_api.cpp:231 Release service-cfg for ps3netsrv when 0x0004 (done)
2018-07-26T01:03:39+02:00 Diskstation916 synodrnode: Delete session of user[sc-ps3netsrv] by user-delete hook



/var/log/synopkg.log:

2018/07/26 18:20:17 install ps3netsrv 1.47.08-2 Begin preinst
/volume1/@tmp/pkginstall/scripts/service-setup: line 20: type: service_preinst: not found
2018/07/26 18:20:17 install ps3netsrv 1.47.08-2 End preinst ret=[0]
2018/07/26 18:20:17 install ps3netsrv 1.47.08-2 Begin /bin/mv -f /volume1/@tmp/pkginstall/package /volume1/@appstore/ps3netsrv
2018/07/26 18:20:17 install ps3netsrv 1.47.08-2 End /bin/mv -f /volume1/@tmp/pkginstall/package /volume1/@appstore/ps3netsrv ret=[0]
2018/07/26 18:20:17 install ps3netsrv 1.47.08-2 Begin /bin/rm -rf /var/packages/ps3netsrv
2018/07/26 18:20:17 install ps3netsrv 1.47.08-2 End /bin/rm -rf /var/packages/ps3netsrv ret=[0]
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 Begin /bin/mkdir -p /var/packages/ps3netsrv
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 End /bin/mkdir -p /var/packages/ps3netsrv ret=[0]
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 Begin /bin/mv -f /volume1/@tmp/pkginstall/INFO /var/packages/ps3netsrv/INFO
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 End /bin/mv -f /volume1/@tmp/pkginstall/INFO /var/packages/ps3netsrv/INFO ret=[0]
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 Begin /bin/rm -rf /var/packages/ps3netsrv/scripts
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 End /bin/rm -rf /var/packages/ps3netsrv/scripts ret=[0]
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 Begin /bin/mv -f /volume1/@tmp/pkginstall/scripts /var/packages/ps3netsrv/scripts
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 End /bin/mv -f /volume1/@tmp/pkginstall/scripts /var/packages/ps3netsrv/scripts ret=[0]
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 Begin /bin/rm -rf /var/packages/ps3netsrv/WIZARD_UIFILES
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 End /bin/rm -rf /var/packages/ps3netsrv/WIZARD_UIFILES ret=[0]
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 Begin /bin/mv -f /volume1/@tmp/pkginstall/WIZARD_UIFILES /var/packages/ps3netsrv/WIZARD_UIFILES
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 End /bin/mv -f /volume1/@tmp/pkginstall/WIZARD_UIFILES /var/packages/ps3netsrv/WIZARD_UIFILES ret=[0]
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 Begin /bin/rm -rf /var/packages/ps3netsrv/conf
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 End /bin/rm -rf /var/packages/ps3netsrv/conf ret=[0]
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 Begin /bin/mv -f /volume1/@tmp/pkginstall/conf /var/packages/ps3netsrv/conf
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 End /bin/mv -f /volume1/@tmp/pkginstall/conf /var/packages/ps3netsrv/conf ret=[0]
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 Begin postinst
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 End postinst ret=[0]
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 Begin /bin/rm -rf /volume1/@tmp/pkginstall
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 End /bin/rm -rf /volume1/@tmp/pkginstall ret=[0]
2018/07/26 18:20:18 install ps3netsrv 1.47.08-2 successfully
2018/07/26 18:20:18 install ps3netsrv: begin to start version 1.47.08-2
2018/07/26 18:20:19 install ps3netsrv 1.47.08-2 Begin pre-load apparmor
2018/07/26 18:20:19 install ps3netsrv 1.47.08-2 End pre-load apparmor ret=[0]
2018/07/26 18:20:19 install ps3netsrv 1.47.08-2 Begin start-stop-status start
/volume1/@appstore/ps3netsrv/var/ps3netsrv.conf: line 2: folder/Games/PS3/GAMES/: No such file or directory
Starting ps3netsrv ...
/var/packages/ps3netsrv/scripts/service-setup: line 20: type: service_prestart: not found
2018/07/26 18:20:19 install ps3netsrv 1.47.08-2 End start-stop-status start ret=[0]
2018/07/26 18:20:19 install ps3netsrv: start version 1.47.08-2 successfully, result 0
2018/07/26 18:20:25 start ps3netsrv: begin to stop version 1.47.08-2
2018/07/26 18:20:25 start ps3netsrv 1.47.08-2 Begin start-stop-status stop
/volume1/@appstore/ps3netsrv/var/ps3netsrv.conf: line 2: folder/Games/PS3/GAMES/: No such file or directory
ps3netsrv is not running


```

### PR DESCRIPTION
**Pull request by Anonym-tsk**
Fix for #3399

### CODE REVIEW

File: spk/ps3netsrv/src/service-setup.sh [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
 
-if [ -r "${CFG_FILE}" ]; then
-    . "${CFG_FILE}"
-fi
+export PID_FILE LOG_FILE CFG_FILE BIN_FILE
 
-SERVICE_COMMAND="/bin/stdbuf -o L -e L ${BIN} ${PS3_DIR}"
-SVC_BACKGROUND=y
-SVC_WRITE_PID=y
+SERVICE_COMMAND="${SYNOPKG_PKGDEST}/ps3netsrv-starter.sh"
```
Comment (ymartin59) [inline spk/ps3netsrv/src/service-setup.sh]: I propose to move `SERVICE_COMMAND` to `Makefile` as other related options `SVC_` vanished.

File: spk/ps3netsrv/src/ps3netsrv-starter.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+if [ -r "${CFG_FILE}" ]; then
+    . "${CFG_FILE}"
+fi
+
+/bin/stdbuf -o L -e L ${BIN_FILE} "${PS3_DIR}" >> ${LOG_FILE} 2>&1 &
```
Comment (ymartin59) [inline spk/ps3netsrv/src/ps3netsrv-starter.sh]: No need to redirection to LOG_FILE as generic script does it already: https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.service.start-stop-status#L39

File: spk/ps3netsrv/src/service-setup.sh [hunk 3]
```diff
@@ -1,13 +1,9 @@
 CFG_FILE="${SYNOPKG_PKGDEST}/var/ps3netsrv.conf"
-BIN="${SYNOPKG_PKGDEST}/bin/ps3netsrv"
+BIN_FILE="${SYNOPKG_PKGDEST}/bin/ps3netsrv"
 
-if [ -r "${CFG_FILE}" ]; then
-    . "${CFG_FILE}"
-fi
+export PID_FILE LOG_FILE CFG_FILE BIN_FILE
```
Comment (ymartin59) [inline spk/ps3netsrv/src/service-setup.sh]: Please move all these variables to your starter script and remove export

Comment (Anonym-tsk) [reply spk/ps3netsrv/src/service-setup.sh]: The starter script doesn't have access to these variables without export.

File: spk/ps3netsrv/src/service-setup.sh (hunk 1 above)
Comment (Anonym-tsk) [reply spk/ps3netsrv/src/service-setup.sh]: Done

File: spk/ps3netsrv/src/ps3netsrv-starter.sh (hunk 2 above)
Comment (Anonym-tsk) [reply spk/ps3netsrv/src/ps3netsrv-starter.sh]: Ok, done

**Comment by ymartin59** [comment]
@Anonym-tsk May you please update to 1.47.09 so that it is worth publishing again. Thank you in advance

**Comment by Anonym-tsk** [comment]
@ymartin59 Done. Please check and merge this PR

File: spk/ps3netsrv/Makefile [hunk 4]
```diff
@@ -1,6 +1,6 @@
 SPK_NAME = ps3netsrv
-SPK_VERS = 1.47.08
-SPK_REV = 2
+SPK_VERS = 1.47.09
+SPK_REV = 1
```
Comment (ymartin59) [inline spk/ps3netsrv/Makefile]: Sorry revision must be 3

File: spk/ps3netsrv/src/wizard/install_uifile_rus [hunk 5]
```diff
@@ -0,0 +1,19 @@
+[{
+    "step_title": "Директория с файлами PS3",
```
Comment (ymartin59) [inline spk/ps3netsrv/src/wizard/install_uifile_rus]: Why not... I should add french translation too then ;)

Comment (Anonym-tsk) [reply spk/ps3netsrv/src/wizard/install_uifile_rus]: French translation already added )

File: spk/ps3netsrv/Makefile (hunk 4 above)
Comment (Anonym-tsk) [reply spk/ps3netsrv/Makefile]: Done

File: spk/ps3netsrv/src/wizard/install_uifile_rus (hunk 5 above)
Comment (PaddyNL) [reply spk/ps3netsrv/src/wizard/install_uifile_rus]: I don’t know this is the right place to ask, if not then I’m sorry about it. Is it possible to give in some more path locations in the ps3netsrv.spk package. In webman you can give in more nethost locations based on Ports 38008, 38009 and so on. On my synology NAS I also have more path’s for PS3 based stuf. Important stuf is on the raid drives les important stuf on a single drive in the system. I have tried to start a second ps3netsrv by command line (SSH) but the applicaties returns with a “error in initialization”

Comment (Anonym-tsk) [reply spk/ps3netsrv/src/wizard/install_uifile_rus]: @PaddyNL At the moment it is impossible. I will try to add this in the future

Comment (PaddyNL) [reply spk/ps3netsrv/src/wizard/install_uifile_rus]: Thx for the quick reply. Then we will have to wait.

**Pull request merged**